src/lib.rs

```diff
@@ -115,47 +109,43 @@ impl Eastern for Chrono {
 
 impl GoldenNumber for Chrono {
     fn get_golden_number(&self) -> u8 {
-        let golden_number: u8 = ((self.year + 1) % 19) as u8;
-        if golden_number == 0 {
-            return 19;
+        match (self.year + 1) % 19 {
+            0 => 19,
+            n => n as u8,
         }
-        golden_number
     }
 }
 
 impl Concurrents for Chrono {
     fn get_concurrents(&self) -> u8 {
         let quarter = self.year / 4;
-        let concurrents: u8 = ((self.year + quarter + 4) % 7) as u8;
-        if concurrents == 0 {
-            return 7;
+        match (self.year + quarter + 4) % 7 {
+            0 => 7,
+            n => n as u8,
         }
-        concurrents
     }
 }
 
 impl SolarCircle for Chrono {
     fn get_solar_circle(&self) -> u8 {
-        let solar_mod: u8 = ((self.year + 9) % 28) as u8;
-        if solar_mod == 0 {
-            return 28;
+        match (self.year + 9) % 28 {
+            0 => 28,
+            n => n as u8,
         }
-        solar_mod
     }
 }
 
 impl Epact for Chrono {
     fn get_epact_j(&self) -> u8 {
-        let golden_number = &self::GoldenNumber::get_golden_number(self);
-        ((golden_number - 1) * 11) % 30
+        ((self.get_golden_number() - 1) * 11) % 30
     }
 
     fn get_epact_g(&self) -> u8 {
         if self.gregorian {
             let century: u8 = ((self.year / 100) + 1) as u8;
             let correction_solar: u8 = (3 * century / 4) as u8;
             let correction_lunar: u8 = ((8.0 * f32::from(century) + 5.0) / 25.0).trunc() as u8;
-            let epact_j: u8 = self::Epact::get_epact_j(self);
+            let epact_j: u8 = self.get_epact_j();
```

Comment:
jakmile voláš metodu s parametrem self, tak ji lze volat přímo na structu / objectu (self) .. a Rust to automaticky doplní jako 1. parametr.